judo-requirement-report-test/src/test/java/hu/blackbelt/judo/requirement/test/TestRequirementProcessor.java

```diff
@@ -84,20 +98,127 @@ public void test04(){
+
+                            @Override
+                            public void printMessage(Diagnostic.Kind kind, CharSequence msg, Element e) {
+                            }
+
+                            @Override
+                            public void printMessage(Diagnostic.Kind kind, CharSequence msg, Element e, AnnotationMirror a) {
+                            }
+
+                            @Override
+                            public void printMessage(Diagnostic.Kind kind, CharSequence msg, Element e, AnnotationMirror a, AnnotationValue v) {
+                            }
+                        };
+                    }
+
+                    @Override
+                    public Filer getFiler() {
+                        return null;
+                    }
+
+                    @Override
+                    public Elements getElementUtils() {
+                        return null;
+                    }
+
+                    @Override
+                    public Types getTypeUtils() {
+                        return null;
+                    }
+
+                    @Override
+                    public SourceVersion getSourceVersion() {
+                        return null;
+                    }
+
+                    @Override
+                    public Locale getLocale() {
+                        return null;
+                    }
+                }
+        );
+
+        // start the process method
+        RuntimeException error = assertThrows(
+                RuntimeException.class,
+                () -> rp.process(
+                        Set.of(new TypeElement[]{}),
+                        new RoundEnvironment() {
+                            @Override
+                            public boolean processingOver() {
+                                return false;
+                            }
+
+                            @Override
+                            public boolean errorRaised() {
+                                return false;
+                            }
+
+                            @Override
+                            public Set<? extends Element> getRootElements() {
+                                return Set.of(new Element[]{});
+                            }
+
+                            @Override
+                            public Set<? extends Element> getElementsAnnotatedWith(TypeElement a) {
+                                return Set.of(new Element[]{});
+                            }
+
+                            @Override
+                            public Set<? extends Element> getElementsAnnotatedWith(Class<? extends Annotation> a) {
+                                return Set.of(new Element[]{});
+                            }
+                        }
+                        )
+        );
+
+        assertNotNull(error);
+        assertThat(error.getMessage(), startsWith("The maven-compiler-plugin doesn't have \"reportPath\" compilerArgs."));
```

Comment:
"The maven-compiler-plugin doesn't have \"reportPath\" compilerArgs." could be extracted as a static final String in RequirementProcessor.java and then could be used here so that changes in exception message could automatically apply for this test as well.